spec.html

```diff
@@ -5546,6 +5794,125 @@ <h1>CopyDataProperties ( _target_, _source_, _excludedItems_ )</h1>
         <p>The target passed in here is always a newly created object which is not directly accessible in case of an error being thrown.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-newprivatename" aoid="NewPrivateName">
+      <h1>NewPrivateName ( _description_ )</h1>
```

Comment:
Is this AO necessary? It's only used in one spot and only has one step.

Reply:
Dropped in https://github.com/tc39/ecma262/pull/1668/commits/f429e6d7c9e9992fad633022fa775c31879d22c6.